mbs-s0014-20230920/README.md

```diff
@@ -0,0 +1,98 @@
+# Map-based noise simulations based on the SAT `mbs-noise-sims-sat` time-domain simulations
+
+Tag: `mbs-s0014-20230920`
+
+## Updates
+
+* 2023-09-20: first release.
+
+## Summary
+
+300 realistic realizations of the SAT map noise, estimated from the [mbs-noise-sims-sat](https://github.com/simonsobs/pwg-scripts/tree/master/pwg-tds/mbs-noise-sims-sat) time-domain noise simulations. Simulations cover the full SAT footprint and utilize the tiled noise model.
+
+## Noise model
+
+This release is based on the [mbs-noise-sims-sat](https://github.com/simonsobs/pwg-scripts/tree/master/pwg-tds/mbs-noise-sims-sat) time-domain simulations, available on `NERSC` at:
+
+    /global/cfs/cdirs/sobs/sims/sat-noise-sims/v1
+
+These time-domain simulations include detector and atmospheric noise scaled to match a full year of sensitivity split into two disjoint maps for each frequency band. 
+
+Using [`mnms`](https://github.com/simonsobs/mnms), we generate 300 realizations of the noise using map-based methods. Specifically, we use the tiled model, governed by the configured [parameters](parameters/so_sat_v1_f1.yaml).
```

Comment:
can you tag a release of `mnms` and mention it here?

Reply:
I have added the tags for `mnms` and `sofind`